lib/autoload/kuroko2/workflow/engine.rb

```diff
@@ -25,7 +25,7 @@ def retry(token)
           node = extract_node(token)
 
           message = "(token #{token.uuid}) Retry current node: '#{node.type}: #{node.option}'"
-          token.job_instance.update_column(:error_at, nil)
+          token.job_instance.update_columns(error_at: nil, retrying: true)
```

Comment:
If some tasks exists, 

eg.
```
my_runner:  first_task
my_runner:  second_task
```

1. a first task has an error.
2. a first task retried and finished successfully.
3. Notified `back to normal`
4. a second task finished successfully
5. Notified `back to normal`

The No.5 notification is not necessary.

Further, the `fork` task and `parallel_fork` task are more complicated. It will not work we want with this implementation.
https://github.com/cookpad/kuroko2/blob/master/docs/tasks.md#fork
https://github.com/cookpad/kuroko2/blob/master/docs/tasks.md#parallel_fork

I think that it is better to set retrying status in `tokens.context`.

Reply:
Thanks, I fixed it in https://github.com/cookpad/kuroko2/pull/93/commits/c5246f2fca1ca870a6ff5043768cb736fca8062b.